src/main/kotlin/com/openlattice/shuttle/destinations/RestDestination.kt

```diff
@@ -19,21 +19,21 @@
  *
  */
 
-package com.openlattice.data.integration.destinations
+package com.openlattice.shuttle.destinations
 
 import com.geekbeast.util.ExponentialBackoff
 import com.geekbeast.util.attempt
 import com.openlattice.data.*
 import com.openlattice.data.integration.Association
 import com.openlattice.data.integration.Entity
-import com.openlattice.data.integration.IntegrationDestination
-import com.openlattice.data.integration.StorageDestination
-import com.openlattice.shuttle.MAX_DELAY
 import java.util.*
 
 /**
  * Writes data using the REST API
  */
+
+const val MAX_DELAY = 8L * 60L * 1000L
```

Comment:
where do these numbers come from?

Reply:
oh is this supposed to be 8 minutes? was confused about the 8. maybe add a comment like `// 8 minutes` just so that people dont have to do math